src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final String DEFAULT_COLOR = Color.WHITE.name();
+    private static final int DEFAULT_RADIUS = 10;
+    private static final int MAX_RANDOM_NUMBER = 100;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public Figure getRandomFigure() {
+        Figure figure;
+        FigureType shape = FigureType.values()[random.nextInt(FigureType.values().length)];
+        String color = colorSupplier.getRandomColor();
+        int firstSize = random.nextInt(MAX_RANDOM_NUMBER);
+        switch (shape) {
+            case CIRCLE:
+                return new Circle(color, firstSize);
```

Comment:
create for each figure a separate method that will be returning a random figure, for example, getRandomSquare() and so on